frontend/packages/ux-editor/src/components/TextResource.module.css

```diff
@@ -32,6 +36,13 @@
   gap: 1rem;
 }
 
+.button {
```

Comment:
if you use `.root > .button` you might not need to use `!important`below

Reply:
It doesn't work because the value it intends to overwrite (in the design system) is also marked with `!important`.

Reply:
😱